clients/src/main/java/org/apache/kafka/common/internals/Topics.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.common.internals;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Common definitions used in client-side tools
+ */
+public final class Topics {
```

Comment:
We should probably add a private constructor to avoid people from instantiating this class. @gwenshap, I suggested the name `Topics` along the lines of `Collections` in Java (there are many such examples in Guava as well). It's a common Java convention to use a plural for a class with utilities and constants. However, I agree that it could be a bit confusing, so your suggestions sound good.